client/layout/main-banner/index.js

```diff
@@ -0,0 +1,31 @@
+import { Gridicon } from '@automattic/components';
+import './style.scss';
+
+export function MainBanner( { sectionName, siteIntent, siteSlug } ) {
+	const allowList = [ 'posts', 'media', 'pages', 'home' ];
+	if ( ! allowList.includes( sectionName ) ) {
+		return null;
+	}
+	return (
+		<div className="layout__launchpad-banner">
+			<div className="layout__launchpad-banner-content">
+				🚀
+				<p>
+					Finish your onboarding in the Launchpad! Return by clicking
```

Comment:
I'm assuming this is intentionally not localized yet, but just wanted to bring it up just in case it is unintentional.

Reply:
Ty -- it was intentional. I should've left notes about this being a WIP since we're exploring different implementations.